tests/python/frontend/tflite/test_forward.py

```diff
@@ -1552,6 +1552,48 @@ def test_forward_squeeze():
     _test_squeeze(np.arange(6).reshape((2, 1, 3, 1)), [1, 3])
 
 
+#######################################################################
+# Quantize/DeQuantize
+# -------------------
+
+def _test_quantize_dequantize(data):
+    """ One iteration of quantize and dequantize """
+
+    import tensorflow as tf2
```

Comment:
I think we don't need this as we have imported in header

Reply:
It's importing tensorflow.compat.v1 as tf. It doesn't have  tf2.lite.TFLiteConverter.from_keras_model

Reply:
How about we use `try` here so that we could prompt users we should have tf 2 so that we could run this test?